the-collab-lab/tcl-26-smart-shopping-list
# Home page and global UI style

## Description

This PR adds CSS for global UI elements (like buttons, link, form fields, etc.) to match our design mockups, as well as style for the home page of the site.

## Related Issue

Closes #73.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
| ✓ | :sparkles: New feature     |
| ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

![image](https://user-images.githubusercontent.com/67769490/131429611-b6faba39-3f6f-4460-af30-7b2428f3c0a5.png)

### After

![image](https://user-images.githubusercontent.com/67769490/131430371-03e56173-583c-45d1-8058-8de432dfb7a9.png)

![image](https://user-images.githubusercontent.com/67769490/131430531-7e8653b3-990e-46e7-a12d-43f57c44e1d9.png)

## Testing Steps / QA Criteria

1. Pull the branch and run npm start.
2. Check that the home page layout looks good in different resolutions.
